cmd/kube-controller-manager/app/plugins_storage_all.go

```diff
@@ -1,3 +1,5 @@
+// +build !skip_storage_plugins_requiring_vendor_api
```

Comment:
sometimes I like to tell little stories in my build tags :)

is there a more succinct tag we can use?

Reply:
@liggitt we went through like 4-5 before ending up here LOL